WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -664,15 +664,23 @@ province_event = {
 					set_character_flag = decided_culture
 				}
 				
-				usurp_title_plus_barony_if_unlanded = PREV
+				ROOT = { county = { usurp_title_plus_barony_if_unlanded = PREVPREV } }
 				set_defacto_liege = THIS
 				reverse_culture = ROOT
 				reverse_religion = ROOT
 				
+				#spawn_unit = { #original code by Zusk, doesn't work properly
+				#	province = PREV
+				#	owner = ROOT
+				#	match_character = ROOT
+				#	match_mult = 0.75
+				#	attrition = 0.000
+				#}
+				# I suggest rewriting the entire unit block
 				spawn_unit = {
-					province = PREV
-					owner = ROOT
-					match_character = ROOT
+					province = ROOT
+					owner = THIS
+					match_character = THIS #not sure who was meant to be here, either the tribal leader or previous holder of the province
```

Comment:
It is the migrating unit which gets troops, if that can help you.

Reply:
Yes, that is actually helpful, since I was also wondering if the troops were supposed to be some kind of garrison moving from the abandoned province.

Reply:
I've chosen the tribal leader as a match character because I thought making the size of the spawning troops depend on the previous owner would make little sense, but the result of using THIS is that the spawned force is rather small.

Reply:
Unless I'm terribly mistaken the idea is that the character that holds the province initiates the migration process and thus takes his tribe with him. The character that will then appear in the province does not get additional troops since he retains those of the tribe that remain in that original province.